### PR TITLE
ICDC-1733

### PR DESCRIPTION
**Pull request by tyudosen**
Changed Linkback URL

### CODE REVIEW

**Review by rana22 — APPROVED** [review]
LGTM

File: src/components/LinkBar/LinkbarView.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { withStyles } from '@material-ui/core';
 const LinkBar = ({ classes }) => (
   <>
     <div className={classes.wrapper}>
-      <a className={classes.link} href="https://datacommons.cancer.gov/?cid=crdcnav_hp_gdc.cancer.gov">NCI Cancer Research Data Commons</a>
+      <a className={classes.link} href="https://datacommons.cancer.gov/?cid=crdcnav_hp_caninecommons.cancer.gov">NCI Cancer Research Data Commons</a>
```
Comment (iksheth) [inline src/components/LinkBar/LinkbarView.js:7]: Looks like this URL dose not exist. Can you confirm with Dimitri?

**Review by iksheth — CHANGES_REQUESTED** [review]

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

File: src/components/LinkBar/LinkbarView.js [hunk 2]
```diff
@@ -1,13 +1,43 @@
+/* eslint-disable react/default-props-match-prop-types */
```
Comment (AjayDoddapaneni) [inline src/components/LinkBar/LinkbarView.js:1]: These need to go away.

File: src/components/LinkBar/LinkbarView.js [hunk 3]
```diff
@@ -29,7 +32,6 @@ const styles = () => ({
     fontFamily: 'Raleway',
     fontSize: '10px',
   },
-
 });
 
 export default withStyles(styles)(LinkBar);
```
Comment (AjayDoddapaneni) [inline src/components/LinkBar/LinkbarView.js:37]: You can use default props in here 
LinkBar.defaultProps = {
  title: 'NCI Cancer Research Data Commons',
  url: 'https://datacommons.cancer.gov/?cid=crdcnav_hp_gdc.cancer.gov',
};

Comment (tyudosen) [reply src/components/LinkBar/LinkbarView.js:39]: Done

**Review by iksheth — APPROVED** [review]

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**